cfgov/data_research/mortgage_utilities/fips_meta.py

```diff
@@ -330,6 +330,50 @@ def load_fips_meta(counties=True):
     load_constants()
 
 
+def load_states():
+    """
+    Load state objects from csv data
+    """
+    State.objects.all().delete()
```

Comment:
You might consider wrapping this method (or even the whole script?) in [an atomic transaction](https://docs.djangoproject.com/en/3.2/topics/db/transactions/#django.db.transaction.atomic):

```py
from django.db import transaction

with transaction.atomic():
    # do stuff
```

That way, if something fails partway through, you're not left in a broken state.

Reply:
Added in a1a015c